### PR TITLE
Added the ability for users to delete their reviews 

### PR DESCRIPTION
**Pull request by HGarcia22**
Created a review index page in React that allows users to view and delete the reviews they have written. We also did some light styling.

### CODE REVIEW

**Comment by HGarcia22** [comment]
Created a review index page in React that allows users to view and delete the reviews they have written. We also did some light styling.

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ReviewsController < ApplicationController
+  def index
+    @reviews = current_user.reviews
+  end
+
+  def destroy
+    binding.pry
```
Comment (CassandraNewell) [inline app/controllers/reviews_controller.rb]: Please remove all debuggers before creating a PR

File: app/views/attractions/index.html.erb [hunk 2]
```diff
@@ -1,18 +1,24 @@
 <div>
   <div class="banner-image">
-  <%= image_tag "BostonTopSpots.jpg" %>
+    <%= image_tag "BostonTopSpots.jpg" %>
+    <div class="bottom-center">
+      <h2>Find Local Attractions, Book Tours,</h2>
+      <h2>Experience Boston</h2>
+      <span class="header-subtitle">
+        blah blahblahblahblabhalbahblabhb lah
```
Comment (CassandraNewell) [inline app/views/attractions/index.html.erb]: Please remove the gibberish!

File: app/controllers/api/v1/reviews_controller.rb [hunk 3]
```diff
@@ -1,5 +1,13 @@
 class Api::V1::ReviewsController < ApplicationController
 
+  def index
```
Comment (CassandraNewell) [inline app/controllers/api/v1/reviews_controller.rb]: Please add a test for these new/modified actions

**Pull request merged**